products/sle/main.pm

```diff
@@ -784,7 +784,12 @@ sub load_consoletests {
     }
     #have SCC repo for SLE product
     if (have_scc_repos()) {
-        loadtest "console/yast_scc";
+        if (get_var("USE_PROXYSCC")) {
+            loadtest "console/proxyscc_postreg";
```

Comment:
We should avoid the name "proxyscc" because the only difference between real SCC and proxy-SCC should be setting the registration url using the variable "SCC_URL". The test module should also work against real SCC and be named accordingly although testing against real SCC should be *nice to have*, not mandatory. To stay consistent, how about "console/suseconnect_scc"?